arangosh/Restore/RestoreFeature.cpp

```diff
@@ -1784,4 +1824,68 @@ Result RestoreFeature::getFirstError() const {
   return {TRI_ERROR_NO_ERROR};
 }
 
+void RestoreFeature::ProgressTracker::updateStauts(const std::string& filename,
```

Comment:
```suggestion
void RestoreFeature::ProgressTracker::updateStatus(const std::string& filename,
```

Reply:
Additionally, I think the first parameter is actually a collection/view name and not a filename.